### ISSUE TITLE
Database is locked - fresh install via pip, python 3.6

### ISSUE THREAD

**Issue by sztomi** [conan-io/conan#803]
I've found a similar issue: https://github.com/conan-io/conan/issues/683

I'm opening this because the stack trace and conditions are a bit different. First of all, python 3.6 is no longer beta, it's the default 3-series installer that python.org offers. I installed conan via pip on a 64-bit Windows 10 VM. After that, I'm getting the following error while running `conan install ..`

```
Traceback (most recent call last):
  File "C:\Users\sztomi\AppData\Roaming\Python\Python36\site-packages\conans\client\store\sqlite.py", line 20, in init
    statement.execute("PRAGMA auto_vacuum = INCREMENTAL;")
sqlite3.OperationalError: database is locked

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\sztomi\AppData\Roaming\Python\Python36\Scripts\conan-script.py", line 11, in <module>
    load_entry_point('conan==0.17.2', 'console_scripts', 'conan')()
  File "C:\Users\sztomi\AppData\Roaming\Python\Python36\site-packages\conans\conan.py", line 6, in run
    main(sys.argv[1:])
  File "C:\Users\sztomi\AppData\Roaming\Python\Python36\site-packages\conans\client\command.py", line 963, in main
    command = get_command()
  File "C:\Users\sztomi\AppData\Roaming\Python\Python36\site-packages\conans\client\command.py", line 950, in get_command
    remote_manager = instance_remote_manager(client_cache)
  File "C:\Users\sztomi\AppData\Roaming\Python\Python36\site-packages\conans\client\command.py", line 914, in instance_remote_manager
    localdb = LocalDB(client_cache.localdb)
  File "C:\Users\sztomi\AppData\Roaming\Python\Python36\site-packages\conans\client\store\localdb.py", line 14, in __init__
    self.init()
  File "C:\Users\sztomi\AppData\Roaming\Python\Python36\site-packages\conans\client\store\localdb.py", line 17, in init
    SQLiteDB.init(self)
  File "C:\Users\sztomi\AppData\Roaming\Python\Python36\site-packages\conans\client\store\sqlite.py", line 22, in init
    raise ConanException("Could not initialize local cache", e)
conans.errors.ConanException: ('Could not initialize local cache', OperationalError('database is locked',))
```

I tried rebooting but that did not help removing the lock. `conan --version` also triggers the same error. Here is the pip version info:

```
> pip show conan
Name: conan
Version: 0.17.2
Summary: Conan C/C++ package manager
Home-page: https://conan.io
Author: JFrog LTD
Author-email: luism@jfrog.com
License: MIT
Location: c:\users\sztomi\appdata\roaming\python\python36\site-packages
Requires: PyJWT, requests, colorama, PyYAML, patch, fasteners, six, node-semver, bottle
```

**Comment by memsharded** [comment]
Ok, if python 3.6 is now the official 3.X release, then certainly this deserve some attention.

I will check in Windows, anyone else feedback of the same behavior in Linux/Mac?

**Comment by memsharded** [comment]
I have been having a look. Something might have changed in the way that python handle open resources, and variables scopes.

I have checked that it can be fixed by adding: 
```python
remote_manager._remote_client._localdb.connection.close()
```
in command.py-L960

Very ugly fix, will try to look for something more elegant

**Comment by tivek** [comment]
Can confirm on up-to-date Arch Linux with Python 3.6.0. I installed conan in a clean virtualenv with `pip install conan` and ran `conan` to get this:
```Traceback (most recent call last):
  File "/home/tivek/projects/python/virtualenvs/conan-py3.6/lib/python3.6/site-packages/conans/client/store/sqlite.py", line 20, in init
    statement.execute("PRAGMA auto_vacuum = INCREMENTAL;")
sqlite3.OperationalError: database is locked

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/tivek/projects/python/virtualenvs/conan-py3.6/bin/conan", line 11, in <module>
    sys.exit(run())
  File "/home/tivek/projects/python/virtualenvs/conan-py3.6/lib/python3.6/site-packages/conans/conan.py", line 6, in run
    main(sys.argv[1:])
  File "/home/tivek/projects/python/virtualenvs/conan-py3.6/lib/python3.6/site-packages/conans/client/command.py", line 963, in main
    command = get_command()
  File "/home/tivek/projects/python/virtualenvs/conan-py3.6/lib/python3.6/site-packages/conans/client/command.py", line 950, in get_command
    remote_manager = instance_remote_manager(client_cache)
  File "/home/tivek/projects/python/virtualenvs/conan-py3.6/lib/python3.6/site-packages/conans/client/command.py", line 914, in instance_remote_manager
    localdb = LocalDB(client_cache.localdb)
  File "/home/tivek/projects/python/virtualenvs/conan-py3.6/lib/python3.6/site-packages/conans/client/store/localdb.py", line 14, in __init__
    self.init()
  File "/home/tivek/projects/python/virtualenvs/conan-py3.6/lib/python3.6/site-packages/conans/client/store/localdb.py", line 17, in init
    SQLiteDB.init(self)
  File "/home/tivek/projects/python/virtualenvs/conan-py3.6/lib/python3.6/site-packages/conans/client/store/sqlite.py", line 22, in init
    raise ConanException("Could not initialize local cache", e)
conans.errors.ConanException: ('Could not initialize local cache', OperationalError('database is locked',))```

**Comment by memsharded** [comment]
Thanks very much for reporting. Working on a fix, will submit PR shortly and will be included in conan 0.18. Also, changing CI (travis/appveyor) to test under python 3.6
